swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/work/WorkQueueManager.kt

```diff
@@ -84,7 +86,16 @@ class WorkQueueManager(
     }
   }
 
-  private fun isEnabled(): Boolean {
+  /**
+   * Swabbie is considered disabled if it is out of service in discovery, disabled via fast property, or the
+   *  schedule determines it is off hours.
+   */
+  internal fun isEnabled(): Boolean {
```

Comment:
Added discovery configuration to this function to make sure swabbie wouldn't work while down.